src/main.js

```diff
@@ -168,7 +166,10 @@ function configureCommandlineSwitchesSync(cliArgs) {
 		'enable-browser-code-loading',
 
 		// Log level to use. Default is 'info'. Allowed values are 'critical', 'error', 'warn', 'info', 'debug', 'trace', 'off'.
-		'log-level'
+		'log-level',
+
+		// Enables render process reuse. Default value is 'true'. See https://github.com/electron/electron/issues/18397
```

Comment:
I would have thought that the default remains `false` and this flag is to enable reuse selectively.

Reply:
Changed feature to opt-in.